docs/src/layouts/base.astro

```diff
@@ -26,6 +26,13 @@ const { title } = Astro.props;
     />
     <title>{`${title === undefined ? "" : `${title} - `}Remeda`}</title>
     <script is:inline src="/dist/scripts/init-theme.js"></script>
+    <script type="module">
+      import * as R from "https://esm.sh/remeda";
+      window.R = R;
+      console.log(
+        "Use variable R right here to access the latest version of the library",
+      );
+    </script>
```

Comment:
Let's make this a little cleaner:

1. Put the script under the public folder (but not under dist/scripts which is for autogenerated scripts).
2. Then, put `<script is:inline fetchpriority="low" src="<location>" type="module"></script>` in the **body** section of the layout, so that the script is only processed *after* the full page loads (as it can wait).

Reply:
I think I did as you suggested. It works but I'm not familiar with astro, please make sure this is what you meant.